assembl/static2/tests/unit/components/common/imageUploader.spec.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import ImageUploader from '../../../../js/app/components/common/imageUploader';
+
+describe('ImageUploader component', () => {
+  it('should render an image uploader', () => {
+    const handleChangeSpy = jest.fn(() => {});
```

Comment:
Ahh `jest` has spying capabilities as well? So, I can assume that we won't be using a system like `sinon.js` to mock/spy objects/methods then...Sinon is what we used in the previous version of Assembl.